Library/UITraitCollection.swift

```diff
@@ -2,19 +2,6 @@ import UIKit
 
 public extension UITraitCollection {
   public func ksr_isAccessibilityCategory() -> Bool {
```

Comment:
We could remove the helper function 🤷‍♂

Reply:
Yep the sole purpose for this was iOS 10 support...so kill it with fire! :)